### PR TITLE
AT 04.00| Folder > Create Folder with a long name

### PR DESCRIPTION
**Pull request by BNastasia**
https://trello.com/c/E2MuZTWL

### CODE REVIEW

File: src/test/java/school/redrover/FolderTest.java [hunk 1]
```diff
@@ -21,6 +21,9 @@ public class FolderTest extends BaseTest {
     private static final String NAME_2 = "Folder";
     private static final String DESCRIPTION = "Created new folder";
     private static final String DISPLAY_NAME = "NewFolder";
+    private static final String LONG_NAME = "qYIs65dT50nvjiognIil5l0c0MxH7PTQZ8enFOY4crE4sb60SPZMt1NgeKQ1nT6P4jgA6R" +
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/FolderTest.java:24]: LONG_NAME used only once, maybe it's better to put it inside the test

File: src/test/java/school/redrover/model/NewJobPage.java [hunk 2]
```diff
@@ -92,4 +92,10 @@ public String getItemNameRequiredErrorText() {
     public String getTitle() {
         return getWait2().until(ExpectedConditions.visibilityOf(title)).getText();
     }
+
+    public CreateBugPage selectJobAndOkAndGoToBugPage(TestUtils.JobType jobType) {
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/model/NewJobPage.java:96]: duplicate of method  selectJobAndOkAndGoError

File: src/test/java/school/redrover/FolderTest.java (hunk 1 above)
Comment (BNastasia) [reply src/test/java/school/redrover/FolderTest.java:24]: done

File: src/test/java/school/redrover/model/NewJobPage.java (hunk 2 above)
Comment (BNastasia) [reply src/test/java/school/redrover/model/NewJobPage.java:96]: that is another page

Comment (evanmaiqa) [reply src/test/java/school/redrover/model/NewJobPage.java:96]: but you know, I'm assuming it's the same page. The "selectJobAndOkAndGoError" method leads to the page "http://localhost:8080/view/all/createItem", just like your method.

**Review by evanmaiqa — APPROVED** [review]

**Review by nikabenz — APPROVED** [review]

**Pull request merged**